tests/phpunit/tests/test-preload-paths.php

```diff
@@ -121,13 +121,11 @@ function ( $taxonomies ) {
 			}
 		);
 
-		$filter_route = new ReflectionClass( PLL_Filter_REST_Routes::class );
-		$get_method   = $filter_route->getMethod( 'get' );
-		$get_method->setAccessible( true );
+		$filter_route  = new PLL_Filter_REST_Routes( $this->pll_admin->model );
+		$preload_paths = $filter_route->add_query_parameters( array( '/wp/v2/genre' ), array( 'test' => 'something' ) );
 
-		$routes = $get_method->invokeArgs( new PLL_Filter_REST_Routes( $this->pll_admin->model ), array() );
-
-		$this->assertArrayHasKey( 'genre', $routes );
-		$this->assertSame( 'wp/v2/genre', $routes['genre'] );
+		// If the parameter is added to the route, this means that the route is one of the filterable routes.
+		$this->assertNotEmpty( $preload_paths );
```

Comment:
```suggestion
		$this->assertNotEmpty( $preload_paths );
		$this->assertCount( 1, $preload_paths );
```
Just to make sure nothing's added.